clients/python/game/api.py

```diff
@@ -0,0 +1,27 @@
+from requests import get
+
+from .models import Game
```

Comment:
[Relative import](https://docs.python.org/2.5/whatsnew/pep-328.html) in the `game` package.